app/controllers/quotes_controller.rb

```diff
@@ -0,0 +1,6 @@
+class QuotesController < ApplicationController
+  def index
```

Comment:
This looks clean. See my comment in your `QuoteFacade`. 
If you were to pass two objects at your facade level, then your controller would look something like:
```
@search_term = params[:query]
    @total_found, @quotes_list = QuotesFacade.quotes(@search_term)
```
Now your view has access to a count and the quote attributes you desire.